# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintJob.h
# Commit: f946921b9173
# Full Hash: f946921b9173529d6ceec8200469747446c98a6a
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2022-05-23 09:52:44
# Regressor Bug: 1770539
# File Overlap Count: 1
# Description:
#   Bug 1770539 p3 - Move nsPrintData::mShrinkRatio to nsPrintJob::mShrinkToFitFactor. r=emilio
#   
#   Depends on D146974
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146975
# ==============================================================================

diff -r 5cd20e839a73 -r f946921b9173 layout/printing/nsPrintJob.h
--- a/layout/printing/nsPrintJob.h	Sun May 22 09:37:56 2022 +0000
+++ b/layout/printing/nsPrintJob.h	Sun May 22 09:37:56 2022 +0000
@@ -260,6 +260,10 @@
   // callback is stored here.
   PrintPreviewResolver mPrintPreviewCallback;
 
+  // The scale factor that would need to be applied to all pages to make the
+  // widest page fit without overflowing/clipping.
+  float mShrinkToFitFactor = 1.0f;
+
   float mScreenDPI = 115.0f;
 
   bool mCreatedForPrintPreview = false;
